conda/plugins/hookspec.py

```diff
@@ -256,6 +258,104 @@ def conda_health_checks():
         """
         yield from ()
 
+    @_hookspec
+    def conda_pre_transactions(self) -> Iterable[CondaPreTransaction]:
```

Comment:
Risking being pedantic here, but these are not "transactions", but "actions". I know it's more verbose, but the technically correct term would be `conda_pre_transaction_actions`. Similarly for the other objects and the `post-` variants.